apps/teams-test-app/src/components/MediaAPIs.tsx

```diff
@@ -8,18 +8,50 @@ const CaptureImage = (): React.ReactElement =>
   ApiWithoutInput({
     name: 'CaptureImage',
     title: 'Capture Image',
-    onClick: async () => {
-      const result = await media.captureImage();
-      const file: media.File = result[0];
-      let content = '';
-      let len = 20;
-      if (file.content) {
-        len = Math.min(len, file.content.length);
-        content = file.content.substr(0, len);
-      }
-      const output =
-        'format: ' + file.format + ', size: ' + file.size + ', mimeType: ' + file.mimeType + ', content: ' + content;
-      return output;
+    onClick: {
+      withPromise: async () => {
+        const result = await media.captureImage();
+        const file: media.File = result[0];
```

Comment:
Can we put that in a helper function outside and reuse in both withPromise and withCallback, to avoid duplication and make sure they are exactly the same (so E2E tests work as expected?)

Reply:
Was thinking the same thing, since the code is the same on both. I'll make that change now.